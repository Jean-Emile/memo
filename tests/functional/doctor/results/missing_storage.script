{
    "drives": [
        {
            "name": "bob/aws_drive",
            "sane": false,
            "volume": {
                "name": "bob/aws_volume",
                "network": {
                    "linked": true,
                    "name": "bob/aws_network",
                    "sane": false,
                    "silos": [
                        {
                            "name": "aws_storage",
                            "reason": "missing",
                            "sane": false,
                            "type": "unknown",
                            "warning": false
                        }
                    ],
                    "warning": false
                },
                "sane": false,
                "warning": false
            },
            "warning": false
        },
        {
            "name": "bob/fs_drive",
            "sane": true,
            "warning": false
        }
    ],
    "leftovers": [],
    "networks": [
        {
            "linked": true,
            "name": "bob/aws_network",
            "sane": false,
            "silos": [
                {
                    "name": "aws_storage",
                    "reason": "missing",
                    "sane": false,
                    "type": "unknown",
                    "warning": false
                }
            ],
            "warning": false
        },
        {
            "linked": true,
            "name": "bob/fs_network",
            "sane": true,
            "warning": false
        }
    ],
    "sane": false,
    "silos": [
        {
            "name": "fs_storage",
            "reason": "\"/tmp/\" is readable and writable",
            "sane": true,
            "type": "filesystem",
            "warning": false
        }
    ],
    "volumes": [
        {
            "name": "bob/aws_volume",
            "network": {
                "linked": true,
                "name": "bob/aws_network",
                "sane": false,
                "silos": [
                    {
                        "name": "aws_storage",
                        "reason": "missing",
                        "sane": false,
                        "type": "unknown",
                        "warning": false
                    }
                ],
                "warning": false
            },
            "sane": false,
            "warning": false
        },
        {
            "name": "bob/fs_volume",
            "sane": true,
            "warning": false
        }
    ]
}
