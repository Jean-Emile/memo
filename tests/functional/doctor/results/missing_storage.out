[OK] User
[OK] Silos
[ERROR] Networks:
  [ERROR] bob/aws_network is faulty because "aws_storage" (missing) silo is faulty
[ERROR] Volumes:
  [ERROR] bob/aws_volume is faulty because network "bob/aws_network" is faulty
[ERROR] Drives:
  [ERROR] bob/aws_drive is faulty because volume "bob/aws_volume" is faulty
[OK] Leftovers
